projects/ngx-location-picker/src/lib/services/ngx-location-picker.helper.ts

```diff
@@ -188,4 +212,24 @@ export class NgxLocationPickerHelper {
     return coordinate;
   }
 
+  /**
+   * Checks wether given coordinates are WGS84 (response true) or Lambert (response false)
+   *
+   * @param x given x coordinate
+   * @param y given y coordinate
+   *
+   * @return isWgs84
+   */
+  isWgs84Coordinates(x?: number, y?: number): boolean {
+    const minX = 49.4;
```

Comment:
isn't it better to check for the -90>90 and -180>180 range? Lambert coordinates are still not going to fall into that range in practice, and it seems more future-proof to check for the full WGS-84 range.

Reply:
We could widen the range, but in the WGS84 coordinate system latitude and longitude are sometimes switched.
The norm is lon/lat (from a mathematical point of view), but lat/lon (as we do) is more common for representation.
So if someone switches the lat/lon, we won’t be able to detect/correct that with the wides range.